### DISCUSSION TITLE
nothing

### DISCUSSION THREAD

**Discussion by liuyongming0612** [GMOD/jbrowse-components#4603] [Q&A] [answered]
https://github.com/GMOD/jbrowse-components/discussions/3143     same question

**Comment by garrettjstevens** [comment, marked as answer]
There is an option called `hideUris` that you can set on a track that prevents the URIs from being shown in the "About track" dialog. Here is an example:

https://github.com/GMOD/jbrowse-components/blob/b78531b75e9d2a10ede46c3fcace8749e43412cd/test_data/volvox/config.json#L648-L670

One important thing to note, though, is that this **does _not_ prevent users from downloading the files**. It hides the URL in the dialog, but they can still find the URL in other ways (e.g. from inspecting the browser's developer tools) and use that to download the file. There is no way to completely prevent users from downloading the files you use with JBrowse.
